src/style.css

```
@@ -1,3 +1,68 @@
+* {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  position: relative;
+}
+
+.logo {
+  padding-top: 2px;
+}
+
+.nav__list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  align-items: center;
+}
+
+.nav__item {
+  margin-right: 20px;
+  position: relative;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
```

1. you can simplify a little
2. item shouldn't be `relative`. Move `position: relative;` to `is-active`
```suggestion
.nav__item:not(:last-child) {
  margin-right: 20px;
}
```